projects/components/src/datagrid/datagrid.component.html

```diff
@@ -9,6 +9,7 @@ <h3 *ngIf="this.header" class="vcd-header">{{ header }}</h3>
     (clrDgRefresh)="gridStateChanged($event)"
     (clrDgSelectedChange)="onClarityDatafridSelectionChange()"
     (clrDgSingleSelectedChange)="onClarityDatafridSelectionChange()"
+    [clrDgPreserveSelection]="preserveSelection"
```

Comment:
Does this mean that we should delete our code that handles `preserveSelection` ?

Reply:
No, our code that handles `preserveSelection` is to clear the selection when the flag is set to `false`. By default the `clr-datagrid` keeps the selection(s) for pagination but sometime we do not want that behavior. Clarity isn't consistent when it comes to filtering because they are clearing the selection(s) on filter change so passing this flag is to the `clr-datagrid` is needed to keep the behavior consistent.